services/dynamic-help-service.ts

```diff
@@ -0,0 +1,26 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import os = require("os");
+
+export class DynamicHelpService implements IDynamicHelpService {
+	constructor(private $injector: IInjector) { }
+	public isProjectType(...args: string[]): IFuture<boolean> {
+		return ((): boolean => {
+			// resolve here as Project.IProject is not part of common-lib
+			var $project: Project.IProject = this.$injector.resolve("project");
```

Comment:
Tns doesn't have `IProject` abstraction. Yep, we're right this is AB CLI specific code.